src/filter.js

```diff
@@ -0,0 +1,15 @@
+export const getFilmsByGenre = (films, genre) => {
+  const allGenres = `All genres`;
+
+  if (genre === allGenres) {
+    return films;
+  }
+
+  let filmsByFilter = films.filter(
```

Comment:
переменная не перезаписывается, нет надобности использовать `let`
```suggestion
  const filmsByFilter = films.filter(
```